README.md

```diff
@@ -32,37 +31,35 @@ years as an in-house engine, used to publish several work-for-hire titles.
 
 #### Binary downloads
 
-Official binaries for the Godot editor and the export templates can be found
+Official binaries for the Godot editor and export templates can be found
 [on the homepage](https://godotengine.org/download).
 
-#### Compiling from source
+#### Compiling Godot from source
```

Comment:
Not sure it's needed here too

Reply:
Yeah - I agree it's inconsistent with the other section headers. I'll remove it.